chevron/renderer.py

```diff
@@ -75,7 +75,7 @@ def _get_key(key, scopes):
             # Return an empty string if falsy, with two exceptions
             # 0 should return 0, and False should return False
             # While using is for this check is undefined it works and is fast
```

Comment:
Should probably remove this comment now that `is` isn't being used anymore?

Reply:
Fixed now. Thanks.

Reply:
Thank you @tirkarthi for doing this PR!